organisms/organisation-unit-tree/new-features/select-descendants.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Explicitly select individual descendant org units inside another org unit
+
+Background:
+  Given an org unit tree with root org unit 'Sierra Leone'
+  And descendants inside 'Sierra Leone' with multiple groups and levels
+
+  Scenario: Selecting all org units of a certain group or level inside another org unit
```

Comment:
Like mentioned above, because the `Background` has a `Given` already, we don't really need the mentioning of context meny here, the `Given` and `When` could be combined into

`When the user select a group or level inside the root org unit`

Reply:
Agreed, less specificity works much better here.